editor/editor_sectioned_inspector.cpp

```diff
@@ -78,15 +78,10 @@ class SectionedInspectorFilter : public Object {
 		for (List<PropertyInfo>::Element *E = pinfo.front(); E; E = E->next()) {
 
 			PropertyInfo pi = E->get();
-			int sp = pi.name.find("/");
 
 			if (pi.name == "resource_path" || pi.name == "resource_name" || pi.name == "resource_local_to_scene" || pi.name.begins_with("script/")) //skip resource stuff
 				continue;
 
-			if (sp == -1) {
-				pi.name = "global/" + pi.name;
```

Comment:
Removing this might break some stuff, I wouldn't merge that for 3.1 at any rate.

Reply:
How? All this does is adding global/ to all project settings without a section.

Reply:
akien expained it to me:
Normally the project settings are supposed to hide/ignore anything that starts with "global/", which is why the sectioned property editor adds it to settings without section.